tile/platform/local_machine/program.cc

```diff
@@ -25,6 +25,12 @@ namespace tile {
 namespace local_machine {
 namespace {
 
+// For Program synchonization
```

Comment:
If they're in an unnamed namespace, they don't need to be static.  (Also, remove the trailing underscores.)
It'd be really nice to put these into a sub-namespace (outside the unnamed namespace), so that accesses would look something like `program_sync::avail_mem`, `program_sync::first_run`, &c.